### PR TITLE
Announce bundle

### PR DESCRIPTION
**Pull request by juliasilge**
This PR adds the announcement blog post for [bundle](https://rstudio.github.io/bundle/).

### CODE REVIEW

**Comment by juliasilge** [comment]
@simonpcouch Can you review this blog post? The [preview is here](https://deploy-preview-593--tidyverse-org.netlify.app/blog/2022/09/bundle-0-1-0/). I was thinking maybe we publish on Friday? But no real rush.

**Review by simonpcouch — COMMENTED** [review]
Love the way you've scoped and pitched this!🙂 Just a few, generally small edits.

If you have the time to make these edits today and feel good about where the post is, you have thumbs up from me on merging today, Friday the 16th, though it looks like the date set for this post is currently Saturday the 17th!

File: content/blog/bundle-0-1-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    predict(model_object, new_data)
+  },
+  args = list(
+    temp_file = temp_file,
+    new_data = head(cell_test)
+  )
+)
+```
+
+Oh no! 😱
```
[the remark this answers is not in the document]
Comment (juliasilge) [reply content/blog/bundle-0-1-0/index.Rmd:155]: Oooof, so glad you caught this 😬

**Comment by juliasilge** [comment]
New [deploy preview](https://deploy-preview-593--tidyverse-org.netlify.app/blog/2022/09/bundle-0-1-0/) is now up!

**Comment by juliasilge** [comment]
> If you have the time to make these edits today and feel good about where the post is, you have thumbs up from me on merging today, Friday the 16th, though it looks like the date set for this post is currently Saturday the 17th!

I absolutely thought it was Wednesday yesterday when I was doing this. 🙈

**Review by simonpcouch — APPROVED** [review]
Yay, looks great to me. :)

**Pull request merged**